templates/BUILD.tpl

```diff
@@ -121,6 +120,16 @@ alias(
 
 ## CC
 
+filegroup(
+    name = "empty",
+    srcs = [],
+)
+
+filegroup(
```

Comment:
Are you keeping this intentionally?

Reply:
Yeah it seems like these 2 filegroups are used by other toolchains here, including the ubuntu gcc and windows toolchains. So I moved them to the top of the file as a "common CC" section.